bin/steps/sqlite3

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set +e
+
+# shellcheck source=bin/utils
+source $BIN_DIR/utils
+
+APT_CACHE_DIR="$CACHE_DIR/.heroku/python/apt/cache"
+APT_STATE_DIR="$CACHE_DIR/.heroku/python/apt/state"
+
+mkdir -p "$APT_CACHE_DIR/archives/partial"
+mkdir -p "$APT_STATE_DIR/lists/partial"
+
+APT_OPTIONS="-o debug::nolocking=true"
+APT_OPTIONS="$APT_OPTIONS -o dir::cache=$APT_CACHE_DIR"
+APT_OPTIONS="$APT_OPTIONS -o dir::state=$APT_STATE_DIR"
+APT_OPTIONS="$APT_OPTIONS -o dir::etc::sourcelist=/etc/apt/sources.list"
+
+puts-step "Installing SQLite3"
+apt-get $APT_OPTIONS update >/dev/null 2>&1
+apt-get $APT_OPTIONS -y -d --reinstall install libsqlite3-dev sqlite3 >/dev/null 2>&1
+
+for deb in $(find $APT_CACHE_DIR/archives/ -name "*.deb"); do
+  dpkg -x $deb "$BUILD_DIR/.heroku/python/sqlite3/"
+done
+
+HEROKU_PYTHON_DIR="$BUILD_DIR/.heroku/python"
+# remove old sqlite3 libraries/binaries
+for header in $(find $HEROKU_PYTHON_DIR/include/ -name "sqlite3*.h"); do
+  rm -f $header
```

Comment:
Is it possible that one of the headers will have a space? Should we quote "$header" ?